projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+Existing projects in Go that can be used as examples.
+
+## HelloFresh
+
+- https://github.com/hellofresh/remote-config-service
```

Comment:
The `readlist` repo is open source, let's remove this link since this is a private repository.

Reply:
@italolelis is this still an issue? ☝️